### ISSUE TITLE
README gives incomplete and outdated instructions

### ISSUE THREAD

**Issue by jsachs** [amida-tech/greyscale#1]
- The README suggests Mongo, but the config (and the code) seem to support Postgres
- Mongoose is used in tests, but is not npm installed.
- Additionally, trying to run the tests immediately yields this error:
```
module.js:338
    throw err;
          ^
Error: Cannot find module 'app/models/user'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/Jacob/Documents/dev/greyscale/backend/test/factories/user.js:1:74)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
```

**Comment by UglyEugen** [comment]
fixed with 3ee1736845d9bc90c352e41898a3c803e72594c9 and 384d95b8164611c4ca00f1e33eeffd9e7da8fe9c

**Comment by UglyEugen** [comment]
and 8d9658d83c895f4d76ced853ed4600b5a49f3805 (DB backup file)
